model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/StaticQueryTests.xtend

```diff
@@ -0,0 +1,244 @@
+package hu.blackbelt.judo.meta.jsl.runtime
+
+import org.eclipse.xtext.testing.InjectWith
+import org.eclipse.xtext.testing.extensions.InjectionExtension
+import org.junit.jupiter.api.^extension.ExtendWith
+import org.eclipse.xtext.testing.util.ParseHelper
+import org.eclipse.xtext.testing.validation.ValidationTestHelper
+import com.google.inject.Inject
+import hu.blackbelt.judo.meta.jsl.jsldsl.ModelDeclaration
+import org.junit.jupiter.api.Test
+import hu.blackbelt.judo.requirement.report.annotation.Requirement
+import org.junit.jupiter.api.Disabled
+
+@ExtendWith(InjectionExtension)
+@InjectWith(JslDslInjectorProvider)
+class StaticQueryTests {
+    @Inject extension ParseHelper<ModelDeclaration>
+    @Inject extension ValidationTestHelper
+
+    @Test
+    @Requirement(reqs = #[
+        "REQ-SYNT-001",
+        "REQ-SYNT-002",
+        "REQ-SYNT-003",
+        "REQ-SYNT-004",
+        "REQ-SYNT-005",
+        "REQ-TYPE-001",
+        "REQ-TYPE-002",
+        "REQ-TYPE-004",
+        "REQ-TYPE-005",
+        "REQ-TYPE-006",
+        "REQ-TYPE-007",
+        "REQ-TYPE-008",
+        "REQ-TYPE-009",
+        "REQ-MDL-001",
+        "REQ-ENT-001",
+        "REQ-ENT-002",
+        "REQ-ENT-004",
+        "REQ-ENT-005",
+        "REQ-ENT-009",
+        "REQ-ENT-011",
+        "REQ-EXPR-001",
+        "REQ-EXPR-002",
+        "REQ-EXPR-004",
+        "REQ-EXPR-006",
+        "REQ-EXPR-007",
+        "REQ-EXPR-008",
+        "REQ-EXPR-022"
+    ])
+    def void tcs018() {
+        '''
+            model modelTCS018;
+            
+            // Decimal
+            type numeric Decimal(precision = 13, scale = 4);
+            
+            type boolean Boolean;
+            type date Date;
+            type time Time;
+            type timestamp Timestamp;
+            type numeric Integer(precision = 9, scale = 0);
+            type numeric Long(precision = 15, scale = 0);
+            type string String(min-size = 0, max-size = 4000);
+            
+            // my enum
+            enum MyEnum {
+                A01 = 1;
```

Comment:
Cryptic enum literals.

Reply:
It has not any meaning. It is used only for the test.